### PR TITLE
Add Auth0

### PR DESCRIPTION
**Pull request by lukeocodes**

### CODE REVIEW

**Comment by mxxcon** [comment]
Is this for when people integrate auth0 on their sites or if I go to https://auth0.com/ and click on login button?

**Comment by mxxcon** [comment]
auth0
Please use this file. It's 1.8kb

File: _data/identity.yml [hunk 1]
```diff
@@ -6,6 +6,19 @@ websites:
       software: Yes
       doc: https://support.1password.com/two-factor-authentication/
 
+    - name: Auth0
+      url: https://auth0.com/
+      img: auth0.png
+      tfa: Yes
+      sms: Yes
+      email: Yes
+      phone: Yes
```
Comment (Carlgo11) [inline _data/identity.yml]: Could you point me to where in the doc phone calls are mentioned?

**Comment by lukeocodes** [comment]
This is for people integrating Auth0 into their site.

File: _data/identity.yml (hunk 1 above)
Comment (lukeocodes) [reply _data/identity.yml]: Just checking on this, I believe through a 3rd party only.

**Comment by mxxcon** [comment]
If it's for people who integrate auth0 into their site then it should not be listed here. See #1177 for justification why.

**Comment by lukeocodes** [comment]
@mxxcon I'm confused, how do we differ from other identity management providers list here: https://twofactorauth.org/#identity

**Comment by Carlgo11** [comment]
@lukeoliff we don't list 2FA providers here (such as Authy, Duo, Google Authenticatior) so if your service is a 2FA provider then it won't be added. (See #1177 like mxxcon mentioned)
Furthermore, if your service is a login service, like using Google, Twitter, Steam or Facebook to login to a website, then it also can't be added. That's because of our third-party rule.

However if your service is like 1Password and Lastpass, allowing the user to store their login information for different things on your service then it's fine.


Are we misinterpreting what your service does?

**Comment by mxxcon** [comment]
@lukeoliff auth0 would be listed in the information you provided was relevant for me going to https://auth0.auth0.com/login and trying to login into auth0 and these are the 2fa methods I could be using.
But not if I'm a webmaster somewhere and want to implement auth0 on my site.

**Pull request closed without merging**